src/MQTT433gateway.cpp

```diff
@@ -178,10 +178,15 @@ void setup() {
 
   settings.onChange(RF_CONFIG, setupRf);
 
-  DebugLn(F("Load Settings"));
+  DebugLn(F("Load Settings..."));
   settings.load();
 
-  DebugLn();
+#ifdef DEBUG
+  DebugLn(F("Current configuration:"));
+  settings.serialize(Serial, true, false);
```

Comment:
How about defining The DEBUG output stream in a constant?

Reply:
See the misplaced comment https://github.com/puuu/MQTT433gateway/pull/21#discussion_r155923463